### ISSUE TITLE
Cannot read property 'mySqlVarLenPgSqlFixedLen' of undefined

### ISSUE THREAD

**Issue by murri71** [AnatolyUss/nmig#23]
Hello, unsure what is going on, but trying a migration of an existing 5.5 mysql db gives
```
/Users/joel/gitstuff/nmig/migration/fmtp/FromMySQL2PostgreSQL.js:198
        } else if ('decimal(19,2)' === mySqlDataType || objDataTypesMap[strDataType].mySqlVarLenPgSqlFixedLen) {
                                                                                    ^

TypeError: Cannot read property 'mySqlVarLenPgSqlFixedLen' of undefined
    at mapDataTypes (/Users/joel/gitstuff/nmig/migration/fmtp/FromMySQL2PostgreSQL.js:198:85)
    at /Users/joel/gitstuff/nmig/migration/fmtp/FromMySQL2PostgreSQL.js:911:69
    at null.<anonymous> (/Users/joel/gitstuff/nmig/node_modules/pg/node_modules/pg-pool/index.js:77:9)
    at Pool.dispense [as _dispense] (/Users/joel/gitstuff/nmig/node_modules/pg/node_modules/pg-pool/node_modules/generic-pool/lib/generic-pool.js:310:14)
    at Pool.release (/Users/joel/gitstuff/nmig/node_modules/pg/node_modules/pg-pool/node_modules/generic-pool/lib/generic-pool.js:429:8)
    at null.<anonymous> (/Users/joel/gitstuff/nmig/node_modules/pg/node_modules/pg-pool/index.js:72:21)
    at null.callback (/Users/joel/gitstuff/nmig/migration/fmtp/FromMySQL2PostgreSQL.js:918:45)
    at Query.handleReadyForQuery (/Users/joel/gitstuff/nmig/node_modules/pg/lib/query.js:114:10)
    at null.<anonymous> (/Users/joel/gitstuff/nmig/node_modules/pg/lib/client.js:172:19)
    at emitOne (events.js:82:20)
    at emit (events.js:169:7)
```

**Comment by AnatolyUss** [comment]
Hello, 
Such an error can occur only if there is some key (MySQL data type) is missing at <code>/nmig/migration/fmtp/DataTypesMap.json</code>...
I honestly tried to find this missing data type, but without success (sorry)... 
Please, have a look at this JSON document, maybe you'll find it.
Sorry again...

**Comment by krowork** [comment]
I found SET type is not set in DataTypesMap.json

**Comment by AnatolyUss** [comment]
Thanks!
I'll update nmig asap!

2016-08-16 14:51 GMT+03:00 krowork <notifications@github.com>:

**Comment by AnatolyUss** [comment]
Hello,
I've released a new version of node (v2.1.0).
Your issue is solved.
Try latest nmig v2.1.0 (if it's still relevant).
Thanks again!
